README.md

```diff
@@ -67,11 +67,40 @@ In the built output you will see these dramatic differences in the bundled code
 
 ### Features scope injection utilities
 
-To handle the difference in scope injection we expose multiple utilities which behave differently per browser in src/utils.js. for Firefox the code exposed handles [xrays correctly](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Sharing_objects_with_page_scripts) without needing the features to be authored differently.
```

Comment:
We should perhaps move all the DDGProxy stuff also then we can remove utils.js from this comment. (Can be a follow up, just thinking outloud)